database/sampleDataQueries.sql

```diff
@@ -32,3 +33,13 @@ VALUES
 INSERT INTO User (FirstName, LastName, EmailAddress, Password, Address, ZipCode, DistancePreference, LastUpdated)
 VALUES
   ('Esther', 'Lin', 'linhsin@oregonstate.edu', 'password', NULL, 10001, NULL, '2022-01-27 12:32:05');
+
+INSERT INTO PetBreed (TypeOfAnimal, Breed)
```

Comment:
What will happen if a user try to save breeds that are not in the database?

Reply:
They should select breeds from the database, I'll change the frontend